File: src/main/java/com/github/cannor147/request/colorization/ColorizationTask.java
package com.github.cannor147.request.colorization;

import com.github.cannor147.painter.RGBColor;
import lombok.Data;
import lombok.RequiredArgsConstructor;

import java.awt.*;
import java.util.List;

@Data
@RequiredArgsConstructor
public class ColorizationTask {
    private final List<Point> points;
    private final RGBColor rgbColor;
    private final boolean onlyPixel;
}

File: src/main/java/com/github/cannor147/geomapper/request/colorization/ColorizationTask.kt
package com.github.cannor147.geomapper.request.colorization

import com.github.cannor147.geomapper.painter.RGBColor
import java.awt.Point

data class ColorizationTask(
    val points: List<Point>,
    val rgbColor: RGBColor,
    val onlyPixel: Boolean = false
)